test/integration/upgrade-command/first-operator/operator.yaml

```diff
@@ -1,5 +1,6 @@
 name: "upgrade-operator"
 version: "0.1.0"
+kubernetesVersion: 1.13
```

Comment:
turns out our test harness is setup with version `1.13.1`

Reply:
I think 1.13 should always mean 1.13 and bigger

Reply:
that is what it means